pkg/api/handlers/projectmcp.go

```diff
@@ -56,7 +56,7 @@ func convertProjectMCPServer(projectServer *v1.ProjectMCPServer, mcpServer *v1.M
 	}
 	pmcp.Alias = mcpServer.Spec.Alias
 
-	if cred != nil && mcpServer.Spec.SharedWithinMCPCatalogName == "" {
+	if mcpServer.Spec.SharedWithinMCPCatalogName == "" {
```

Comment:
Note: I removed the `cred != nil` check because it was preventing the fields below from being set for "copied" mcp servers